depot/signer.go

```diff
@@ -78,6 +89,11 @@ func (s *Signer) SignCSR(m *scep.CSRReqMessage) (*x509.Certificate, error) {
 		return nil, err
 	}
 
+	signatureAlgo := m.CSR.SignatureAlgorithm
```

Comment:
Let's just not set this at all/leave it default. In this way it'll default to the to SHA256 for RSA keys (which is what the CA key is, by default).

```suggestion
	var signatureAlgo x509.SignatureAlgorithm
```

Reply:
@jessepeterson  sounds good. Will change - do you still want to keep the override i'm assuming ?